src/app/router.tsx

```
@@ -34,6 +35,14 @@ const Router = createBrowserRouter([
     path: "/product/:id/party/:partyId",
     element: <ProductDetail />,
   },
+  {
+    path: "auth/login",
+    element:<Login />,
```

ESLint

[error] Insert `·`

(prettier/prettier)